build/style_test.go

```diff
@@ -502,12 +502,13 @@ func TestStyle(t *testing.T) {
 			}
 			return nil
 		})
+		settingsPkg := `github.com/cockroachdb/cockroach/pkg/settings`
```

Comment:
looks like the reusable bit is really `settingsPkg+": "`, maybe make that the extracted string and call it settingsPkgImports or some such